<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Body</class>
 <widget class="QWidget" name="Body">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>1200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1140</y>
     <width>200</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	background-color: rgb(255, 255, 255);
font: 75 13pt &quot;微软雅黑&quot;;
border-radius:75px;
border:8px grey;
border-style:inset;}
</string>
   </property>
   <property name="text">
    <string>得分：0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>500</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="title">
    <string>游戏时间设定</string>
   </property>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px grey;

background-color: rgb(255, 255, 255);
border-style:inset;
font: 75 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 11pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>s</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>107</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px grey;
font: 75 11pt &quot;黑体&quot;;
border-style:inset;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
